src/arns.lua

```diff
@@ -296,27 +305,24 @@ function arns.assertValidBuyRecord(name, years, purchaseType, processId)
 	end
 
 	-- assert processId is valid pattern
-	assert(type(processId) == "string", "ProcessId is required and must be a string.")
-	assert(utils.isValidBase64Url(processId), "ProcessId pattern is invalid.")
+	assert(type(processId) == "string", "Process id is required and must be a string.")
+	assert(utils.isValidAOAddress(processId), "Process id must be a valid base64url.")
```

Comment:
`a valid base64url` is somewhat ambiguous. Consider: `must be a valid AO signer address.`

Reply:
tweaked here - [8550415](https://github.com/ar-io/ar-io-network-process/pull/161/commits/85504157f932d851172d0d028ec06f66be7f244e)